scripts/34-Plain-Text-Team.md

```diff
@@ -91,20 +91,20 @@ notes:
 
 # Tie Yourself to the Mast
 
-In the Odyssey, Ulysses had to travel through siren-infested waters.
+In the Odyssey, Odysseus (confusingly called Ulysses in English literature) had to travel through siren-infested waters.
```

Comment:
@nottrobin here's my commit with the Ulysses fixes, thank you for pointing this out

Reply:
This is perfect, thanks :)